packages/mobile/src/helpers/permissions/permissions.ts

```diff
@@ -0,0 +1,25 @@
+import { request, PERMISSIONS, RESULTS } from 'react-native-permissions';
+import { notification } from '~/services/services';
+
+const requestExternalStoragePermission = async () => {
+  try {
+    const grantedGallery = await request(
+      PERMISSIONS.ANDROID.WRITE_EXTERNAL_STORAGE,
+      {
+        title: 'App Gallery Permission',
+        message: 'App needs access to your photos',
+        buttonNeutral: 'Ask Me Later',
+        buttonNegative: 'Cancel',
+        buttonPositive: 'OK',
+      },
```

Comment:
We may skip it and leave up to the device to handle correct language

Reply:
updated